include/mraa_internal_types.h

```diff
@@ -454,6 +454,16 @@ typedef struct {
     /*@}*/
 } mraa_aio_dev_t;
 
+/**
+ * A Structure representing a LED device.
```

Comment:
style: no need in the first "A" and the second one should be "an" to be aligned with the style you've used elsewhere

Reply:
Ack